README.rst

```diff
@@ -1629,6 +1632,15 @@ Get device group detail
 
     GET /api/v1/controller/group/{id}/
 
+Patch device group detail
+#########################
+
+.. code-block:: text
+
+    PATCH /api/v1/controller/group/{id}/
```

Comment:
please let's mention put here and "change" in the heading, I am not sure why some endpoints have patch instead of PUT but let's use PUT here as it makes more sense.